_posts/fr/newsletters/2022-09-14-newsletter.md

```diff
@@ -0,0 +1,171 @@
+  synchronisation d'en-têtes 'initial'. Mais si, pendant la synchronisation des en-têtes, un noeud
+  envoie un message `inv` qui annonce des blocs, alors sans ce PR, le noeud commencera à demander des
+  en-têtes à ce pair aussi, _sans_ mettre son flag `fSyncStarted`. Ceci est la source des messages
+  d'en-têtes redondants, et n'était probablement pas prévu, mais a l'avantage de permettre à la
+  synchronisation des en-têtes de continuer même si le pair initial de synchronisation des en-têtes est
+  malveillant, cassé, ou très lent. Avec ce PR, le noeud demande des en-têtes à seulement _un_ pair
+  supplémentaire (plutôt qu'à tous les pairs qui ont annoncé le nouveau bloc)."
+  a3link="https://bitcoincore.reviews/25720#l-102"
+
+  q4="Une alternative à l'approche adoptée dans le PR serait d'ajouter un pair de synchronisation
+  d'en-têtes supplémentaire après un délai d'attente (fixe ou aléatoire). Quel est l'avantage
+  de l'approche adoptée dans le PR par rapport à cette alternative ?"
+  a4="Un avantage est que les pairs qui nous annoncent un `inv` ont une plus grande probabilité
+  d'être réactifs. Un autre avantage est qu'un pair qui réussit à nous envoyer le bloc `inv`
+  en premier est souvent aussi un pair très rapide. Nous ne choisirons donc pas un autre pair
+  lent si, pour une raison quelconque, notre pair initial est lent."
+
+  q4link="https://bitcoincore.reviews/25720#l-135"
+%}
+
+## Mise à jour et candidat à la publication
+
+*Nouvelles mises à jour et candidat à la publication pour le projet principal d'infrastructure
+de Bitcoin. S'il vous plait prévoyez de mettre à jour à la nouvelle version ou d'aider
+à tester le candidat à publication.*
+
+- [LDK 0.0.111][] ajoute un support pour la création, la réception et la relayage
+  [onion messages][topic onion messages], parmi plusieurs autres nouvelles fonctionnalités
+  et de corrections de bogues.
+
+## Changements notables dans le code et la documentation
+
+*Changements notables cette semaine dans [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], et [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25614][] construit sur [Bitcoin Core #24464][], permettant
+  la possibilité d'ajouter et de tracer des journaux avec des niveaux spécifiques
+  de gravité dans addrdb, addrman, banman, i2p, mempool, netbase, net, net_processing,
+  timedata, et torcontrol.
+
+- [Bitcoin Core #25768][] corrige un bogue où le portefeuille ne rediffusait
+pas toujours les transactions enfants des transactions non confirmées.
+Le portefeuille intégré de Bitcoin Core tente périodiquement de diffuser l'une
+de ses transactions qui n'a pas encore été confirmée. Certaines de ces
```

Comment:
I believe it's not one but any (i.e. all) of its unconfirmed transactions.

Reply:
Hello, I did all modification, if you can check one more time,
Sorry for this supplementary job,